app/controllers/application_controller.rb

```diff
@@ -3,4 +3,11 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
   include SessionsHelper
+  
+  
+  # Confirms an admin employee.
+    def admin_employee
```

Comment:
As a clarification point, will this always be the consistent behavior if the employee is not admin? If so, have it reflect in the method name itself (e.g. redirect_to_root_unless_admin).